src/common/search/search-input.ts

```diff
@@ -38,7 +41,7 @@ class SearchInput extends LitElement {
       </style>
       <paper-input
         class=${classMap({ "no-underline": this.noUnderline })}
-        autofocus
+        .autofocus=${this.autofocus}
```

Comment:
Now we have changed this, we should adjust it on the other places we use `search-input`